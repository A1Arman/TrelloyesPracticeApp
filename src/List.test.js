import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import List from './List';


describe('List component', () => {
    it('renders without crashing', () => {
        const section = document.createElement('section');
        ReactDOM.render(<List cards={[ ]}/>, section);
        ReactDOM.unmountComponentAtNode(section);
    });
    
    it('Testing the List output', () => {
        const tree = renderer
            .create(<List header='Hello' cards={[ ]} />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});




