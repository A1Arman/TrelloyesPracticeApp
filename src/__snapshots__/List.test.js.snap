// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List component Testing the List output 1`] = `
<section
  className="List"
>
  <header
    className="List-cards"
  >
    <h2>
      Hello
    </h2>
  </header>
  <div
    className="List-cards"
  >
    <button
      className="List-add-button"
      type="button"
    >
       + Add a Card
    </button>
  </div>
</section>
`;
